tests/acceptance/stepDefinitions/provisioningContext.js

```diff
@@ -47,7 +49,7 @@ function createUser (userId, password, displayName = false, email = false) {
           const body = new URLSearchParams()
           body.append('key', 'email')
           body.append('value', email)
-          fetch(`${client.globals.backend_url}/ocs/v2.php/cloud/users/${encodeURIComponent(userId)}?format=json`,
+          fetch(`${backendHelper.getCurrentBackendUrl()}/ocs/v2.php/cloud/users/${encodeURIComponent(userId)}?format=json`,
```

Comment:
Why doesn't `join` work here, but works in line70?

Reply:
no idea. I have created new issue for checking this kinds of things in all places https://github.com/owncloud/phoenix/issues/2649